### PR TITLE
CVF-13: Index some creation parameters

### PR DESCRIPTION
**Pull request by kmbarry1**
IMO `tok0` and `tok1` make the most sense to index, although I could possibly see an argument for `sender`--but I generally expect that to be low cardinality, unless other teams besides Maker start using this exact same oracle code. I don't think `orcl` is very interesting to index if `tok0` and `tok1` are indexed. `orb0` and `orb1` are probably more interesting to have as data to check if they are what is expected based on `tok0` and `tok1`. I think indexing `wat` likely becomes redundant as well if `tok0` and `tok1` are indexed.

### CODE REVIEW

File: src/UNIV2LPOracle.t.sol [hunk 1]
```diff
@@ -331,7 +331,9 @@ contract UNIV2LPOracleTest is DSTest {
 
     // Passed 10 rounds of fuzzing with 10,000 test cases
     function test_compare_sqrt(uint256 exp) public {
-            // Convert to WAD since that is what we operate on
+        if (exp == 0) return;
```
Comment (WilfredTA) [inline src/UNIV2LPOracle.t.sol:334]: Good addition here heh

**Review by WilfredTA — APPROVED** [review]
Also agree tokens indexed are most useful.

File: src/UNIV2LPOracle.t.sol (hunk 1 above)
Comment (kmbarry1) [reply src/UNIV2LPOracle.t.sol:334]: This edge case was actually making the tests flaky...so I snuck it into this PR ;).
